src/factories/ModuleFactory.sol

```diff
@@ -28,8 +30,8 @@ import {
  *
  * @dev An owned factory for deploying modules.
  *
- *      The owner can register module metadata's to an {IBeacon}
- *      implementations. Note that a metadata's registered {IBeacon}
+ *      The owner can register module metadata's to an {IInverterBeacon}
+ *      implementations. Note that a metadata's registered {IInverterBeacon}
```

Comment:
Pedantic-Mode still active: Wouldn't it make more sense to refer to it as `InverterBeacon` here? As users are not interacting **with** the interface but just **through** it. You register the metadata to an InverterBeacon, not it's interface. Might just be me though :D

Or are you calling it `IInverterBeacon` here because you refer to an instance of the interface?

Reply:
![giphy](https://github.com/InverterNetwork/inverter-contracts/assets/47272854/b22f520a-c923-46ba-8a67-0eb2e653d0a8)

Honestly I just replaced the original IBeacon here that Pascal initially wrote xD
I dont mind changing it

Reply:
"You register the metadata to an InverterBeacon, not it's interface."
-> The factory actually registers only the interface not the InverterBeacon. As long as the interface is fulfilled anything can be hidden behind there